operator/builtin/parser/keyvalue/testdata/delimiter.yaml

```diff
@@ -0,0 +1,2 @@
+type: key_value_parser
+delimiter: ";"
```

Comment:
```suggestion
delimiter: ";"

```

Reply:
We're not consistent on this in this library, but we need to move in that direction.